front/lib/swr/data_source_views.ts

```diff
@@ -160,6 +161,101 @@ export function useDataSourceViewContentNodes({
   };
 }
 
+export function useDataSourceViewContentNodesWithInfiniteScroll({
+  owner,
+  dataSourceView,
+  internalIds,
+  parentId,
+  pageSize = 50,
```

Comment:
Pagination is not supported on managed data source, what's the plan there?

Reply:
Pagination is supported by content-nodes api , even if it's "faked" pagination for managed data-sources for now ( this has been fixed here : https://github.com/dust-tt/dust/pull/7343 - the fact of having pagination working for static and whole result returned for managed while still passing pagination params was breaking datatable pagination).
I still think we will have to implement it in connectors at some point (better sooner than later)

Reply:
Oh didn't know.